### PR TITLE
docs - pxf troubleshooting - address jdbc timezone error

### PR DESCRIPTION
**Pull request by lisakowen**
add section to pxf troubleshooting page addressing timezone errors with jdbc connector.

i assumed only the jdbc connector could run into this problem.  let me know if that is not the case. 
 also, wasn't sure if i characterized the source of the mismatch correctly.

note:  this PR does not include doc changes for the new pxf internal/user configuration directory file split.  those changes will be addressed in a subsequent PR.

### CODE REVIEW

File: gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb [hunk 1]
```diff
@@ -198,3 +198,23 @@ Perform the following procedure to decrease the maximum number of Tomcat threads
     $ gpadmin@gpmaster$ gpssh -e -v -f seghostfile "/usr/local/greenplum-db/pxf/bin/pxf restart"
     ```
 
+## <a id="pxf-timezonecfg"></a>Addressing PXF JDBC Connector Time Zone Errors
+
+You use the PXF JDBC connector to access data stored in an external SQL database. Depending upon the JDBC driver, PXF may return an error if there is a mismatch between the default time zone set for the PXF server and the time zone set for the external SQL database.
+
```
Comment (lavjain) [inline gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: The error is specific to JDBC driver (not to do anything with PXF).

File: gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb [hunk 2]
```diff
@@ -198,3 +198,23 @@ Perform the following procedure to decrease the maximum number of Tomcat threads
...
+SEVERE: Servlet.service() for servlet [PXF REST Service] in context with path [/pxf] threw exception
+java.io.IOException: ORA-00604: error occurred at recursive SQL level 1
+ORA-01882: timezone region not found
+```
+
+Should you encounter this error, you can set a default time zone for the PXF server in the `$GPHOME/pxf/conf/pxf-env.sh` configuration file, `PXF_JVM_OPTS` property setting. For example:
+
+``` pre
+export PXF_JVM_OPTS="<current_settings> -Duser.timezone=America/Chicago"
+```
```
Comment (lavjain) [inline gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: The user can set any additional java options here as well e.g.` -Duser.country=CA -Duser.language=fr` based on the error message that they are getting when connecting to the external datasource.

File: gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb [hunk 3]
```diff
@@ -198,3 +198,23 @@ Perform the following procedure to decrease the maximum number of Tomcat threads
...
+
+For example, if you use the PXF JDBC connector to access an Oracle database with a conflicting time zone, PXF logs an error similar to the following:
+
+``` pre
+SEVERE: Servlet.service() for servlet [PXF REST Service] in context with path [/pxf] threw exception
+java.io.IOException: ORA-00604: error occurred at recursive SQL level 1
+ORA-01882: timezone region not found
+```
+
+Should you encounter this error, you can set a default time zone for the PXF server in the `$GPHOME/pxf/conf/pxf-env.sh` configuration file, `PXF_JVM_OPTS` property setting. For example:
```
Comment (lavjain) [inline gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: The location for pxf-env.sh is $PXF_CONF/conf which is different than $PXF_HOME/conf

Comment (lisakowen) [reply gpdb-doc/markdown/pxf/troubleshooting_pxf.html.md.erb]: yes, lav.  i am submitting a separate PR for the changes related to the pxf internal/user config directory split.  (this page will have other updates related to that work.)    i will make sure to identify that the user should modify $PXF_CONF/conf/pxf-env.sh.

**Pull request merged**